lib/Storage/MongoDbTimeSeriesStore.php

```diff
@@ -569,6 +570,11 @@ public function getStats($filterBy = [])
             $match['$match']['parentCampaignId'] = $parentCampaignId;
         }
 
+        // Has Parent Campaign Filter
+        if ($hasParentCampaign) {
```

Comment:
I don't think this filter is name correctly, because if you provided false to this you might expect to get back records without a parentCampaign only. - would a better name be `mustHaveParentCampaign` ?